### PR TITLE
✨Adobe XD support

### PR DESCRIPTION
**Pull request by thibmaek**
Documentation: https://www.notion.so/thibmaek/Adobe-XD-b67717dbef514c139c557f0f60400b93

## Checklist for this feature:
- [x] Complies to Phase 1 of [adoption lifecycle](https://www.notion.so/thibmaek/When-is-a-design-tool-considered-compatible-0742982d23274834b38076abec740ed4)
- [x] Has unit test / integration test
- [x] Output is as close as possible to that of other design parsers
- [x] Has edge cases or relevant documentation added to the documentation (see above)
- [x] All currently [supported tokens](https://www.notion.so/thibmaek/5e248fa901c4467a8f39d9eaef75b755?v=f44221778a3344fa87758e3b6cfc95d9) are extracted, unless impossible

### CODE REVIEW

**Comment by thibmaek** [comment]
@robbedec when we merge this new feature, can you review the checklist and merge develop again

**Comment by thibmaek** [comment]
@robbedec after your approval I'll merge

File: __tests__/verifyMocks.test.js [hunk 1]
```diff
@@ -39,4 +43,16 @@ describe('Verify that the mocks are unchanged', () => {
       expect(styleDictFigma).toMatchSnapshot();
     });
   });
+
+  describe('for Adobe XD', () => {
+    it.skip('-> Raw dump', () => {
```
Comment (robbedec) [inline __tests__/verifyMocks.test.js:48]: Figma and Sketch also contain a snapshot test for the sample dump (also skipped). Can we not remove them?

File: lib/mappers/adobexd/blur.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const { tokenize } = require('../../utils');
+
+function mapBlur(artboard) {
+  const { params } = artboard.value.style.filters[0];
+
+  return {
+    id: artboard.name,
```
Comment (robbedec) [inline lib/mappers/adobexd/blur.js]: camelcase name

File: lib/utils.js [hunk 3]
```diff
@@ -51,6 +57,17 @@ module.exports = {
     return [...prev, curr];
   }, []),
 
+
```
Comment (robbedec) [inline lib/utils.js]: Remove extra whitespace

**Review by robbedec — APPROVED** [review]
